detox/src/devices/Device.js

```diff
@@ -180,15 +184,10 @@ class Device {
     await this.deviceDriver.setLocation(this._deviceId, lat, lon);
   }
 
-  //Do not remove this, it is very important for our customers!
-  async __debug_sleep(ms) {
-    return new Promise(resolve => setTimeout(() => resolve(), ms));
-  }
-
   async _sendPayload(key, params) {
     const payloadFilePath = this.deviceDriver.createPayloadFile(params);
     let payload = {};
-    //JS does not support {key: "asd"} JSON generation where the `key` is a variable 🤦‍♂️
+    //JS does not support {key: "asd"} JSON generation where the `key` is a variable
```

Comment:
🤦‍♂️

Reply:
Not done yet...
+ There's Android client to align to the protocol changes